Library/Homebrew/cmd/update-report.rb

```diff
@@ -135,6 +137,7 @@ def output_update_report
       if old_tag.blank? || (new_tag == old_tag)
         puts "Updated Homebrew from #{shorten_revision(initial_revision)} to #{shorten_revision(current_revision)}."
       else
+        Settings.write "latesttag", new_tag
```

Comment:
```suggestion
      Settings.write "latesttag", new_tag unless new_tag == old_tag

      if old_tag.blank? || (new_tag == old_tag)
        puts "Updated Homebrew from #{shorten_revision(initial_revision)} to #{shorten_revision(current_revision)}."
      else
```

Currently, if `old_tag` is blank (i.e. never been written before), the new tag will never be written since the `if` condition will never be false.

Reply:
I don't understand why this was moved from the `Settings.write "latesttag", new_repository_version` location below.

Reply:
@Rylan12 ` if old_tag.blank? ` is redundant when we evaluate for `new_tag == old_tag` too.

@MikeMcQuaid it was moved to avoid being bypassed by `return if Homebrew::EnvConfig.disable_load_formula?` and now `return if new_tag.blank? || args.quiet?`. I also think it is clearer having it closer to the read and short update message.